packages/embed/src/initializers/initialize-sidetabs.ts

```diff
@@ -0,0 +1,23 @@
+import { createSidetab } from '../factories/create-sidetab'
+import { includeCss } from '../utils'
+import { SIDETAB_ATTRIBUTE } from '../constants'
+
+import { buildOptionsFromAttributes } from './build-options-from-attributes'
+
+export const initializeSidetabs = () => {
+  const sidetabElements = document.querySelectorAll<HTMLElement>(`[${SIDETAB_ATTRIBUTE}]`)
+
+  if (sidetabElements.length) {
+    includeCss('./lib/css/sidetab.css')
```

Comment:
Method [includeCss](https://github.com/Typeform/embed/blob/6d7c673b0694318525c5b9774f885c319f29eb10/packages/embed/src/utils/include-css.ts#L3) takes just the filename, not whole path.